src/renderer/services/clients/types.ts

```diff
@@ -37,15 +38,15 @@ export type LoadTxsParams = {
   offset: number
 }
 
-export type TxsParams = { asset: O.Option<Asset> } & LoadTxsParams
+export type TxsParams = { asset: O.Option<Asset>; walletAddress: O.Option<string> } & LoadTxsParams
 
 export type TxsPageRD = RD.RemoteData<ApiError, TxsPage>
 export type TxsPageLD = LiveData<ApiError, TxsPage>
 
-export type BalanceRD = RD.RemoteData<ApiError, Balance>
+export type BalanceRD = RD.RemoteData<ApiError, WalletBalance>
 
-export type BalancesRD = RD.RemoteData<ApiError, Balances>
-export type BalancesLD = LiveData<ApiError, Balances>
+export type BalancesRD = RD.RemoteData<ApiError, WalletBalance[]>
+export type BalancesLD = LiveData<ApiError, WalletBalance[]>
```

Comment:
How about to rename ^`

`BalancesRD` ->  `WalletBalancesRD`
`BalancesLD` ->  `WalletBalancesLD`
?